test/mocha.opts

```diff
@@ -0,0 +1,5 @@
+--require ./node_modules/ts-node/register	
+--require ./node_modules/source-map-support/register	
+--recursive	
+test/**/*.ts	
+
```

Comment:
Mocha opts is deprecated and was replaced by the .mocharc.js

https://mochajs.org/#mochaopts

Reply:
Agree this change shouldn't be part of this PR. 

This change broke VSCode's test explorer and I'm not sure why.  I'm rolling it back here: https://github.com/xpring-eng/Xpring-JS/pull/182